packages/libsql-core/src/uri.ts

```diff
@@ -90,7 +90,7 @@ const AUTHORITY_RE = (() => {
     const USERINFO = '(?<username>[^:]*)(:(?<password>.*))?';
```

Comment:
A suggestion here to make this more readable, let me know if this looks good I can add this here.

```suggestion
    const USERINFO = "((?<username>[^:]*)(:(?<password>.*))?@)?";
    const HOST = "((?<host>[^:\\[\\]]*)|(\\[(?<host_br>[^\\[\\]]*)\\]))";
    const PORT = "(:(?<port>[0-9]*))?";
    return new RegExp("^" + USERINFO + HOST + PORT + "$", "su");
```